styleguide/components/41-organisms/footer/footer.twig

```diff
@@ -1,4 +1,4 @@
-<footer class="footer" role="contentinfo">
+<footer class="footer">
```

Comment:
Role contentinfo was actually correct.
The error was caused because it was nested inside a main tag (which is not correct).

Reply:
Due to the template preview in Fractal this will always give an error I think... :(

Reply:
Fixed it.